### ISSUE TITLE
Bug Report: app v. 0.5.23.0 - Strings untranslated 

### ISSUE THREAD

**Issue by bovirus** [Timthreetwelve/WUView#46]
### Is there an existing issue for this?

- [X] I have searched the existing issues and have read the Known Issues and Troubleshooting topics in the Wiki

### What happened or didn't happen

@Tim

Please run the app.
Check screenshot

![image](https://github.com/Timthreetwelve/WUView/assets/1262554/836da5d5-4df4-4357-80bb-504525b724a0)

The string "Succeeded" and "Failed" are untranslated.

### Version

0.5.23.0

### Windows Version

Windows 11 22H2 64bit italian

**Comment by Timthreetwelve** [comment]
@bovirus, the result codes are in English in the [enumeration](https://learn.microsoft.com/it-IT/windows/win32/api/wuapi/ne-wuapi-operationresultcode#constants). I had just assumed that since it's part of Microsoft's code it would have been translated to the current culture. If you could supply a translation for the six values in the enum I'll see about writing a method to replace them.

Also, if it's not too much trouble, would you attach a screenshot with an update selected so that the details panel on the bottom is populated?

**Comment by bovirus** [comment]
@Timthreetwelve 

Sure. Done

![image](https://github.com/Timthreetwelve/WUView/assets/1262554/3d383f07-6626-45dc-9607-fbf72eb99779)

**Comment by Timthreetwelve** [comment]
Thanks for the screenshot. Even though I can change language in the app, the details (title, description, etc.) are still in English for me.

I've added logic and strings to localize the operation type (Install or Uninstall). There are just two strings:

``` XML
    <!--  Update Operation Type  -->
    <sys:String x:Key="OperationType_Installation">Installation</sys:String>
    <sys:String x:Key="OperationType_Uninstallation">Uninstallation</sys:String>
```

Thanks again for your help.

Tim

**Comment by bovirus** [comment]
@Timthreetwelve 

I updated italian translation

https://github.com/Timthreetwelve/WUView/pull/50

**Comment by Timthreetwelve** [comment]
Thank you.

**Comment by Timthreetwelve** [comment]
This issue should be resolved as of version 0.5.24.
